kinto/core/storage/memory.py

```diff
@@ -354,7 +353,29 @@ def get_all(
             pagination_rules=pagination_rules,
             limit=limit,
         )
-        return objects, count
+        return objects
+
+    @deprecate_kwargs({"collection_id": "resource_name"})
```

Comment:
We are not supposed to have deprecate stuff for methods that were just introduced. I guess there is a path somewhere that makes a call to this `count_all` method with the wrong parameter...  

IMO the caller should be in charge of rebinding the old `collection_id` parameter to `resource_name`, but I guess we could avoid having this here!